src/Parser/RawBody.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace React\Http\Parser;
+
+use Evenement\EventEmitterTrait;
+use React\Http\Request;
+
+class RawBody implements ParserInterface
+{
+    use EventEmitterTrait;
+
+    protected $buffer = '';
+    protected $contentLength;
+
+    public function __construct(Request $request)
+    {
+        $headers = $request->getHeaders();
+        $headers = array_change_key_case($headers, CASE_LOWER);
+
+        $this->contentLength = $headers['content-length'];
```

Comment:
> It isn't and would be a nice bonus, perhaps finishing this PR and then do a followup adding that?

Strong +1. As php-pm user I control client and server both and am really interested in getting volkszaehler/httpd to install at all without messing with unstable dependencies. Anything that will get 0.5 out of the door would be highly appreciated.